Development/ids.xsd

```diff
@@ -229,12 +243,30 @@
 	</xs:complexType>
 	<xs:simpleType name="relations">
 		<xs:restriction base="xs:string">
-			<xs:enumeration value="IFCRELAGGREGATES"/>
-			<xs:enumeration value="IFCRELASSIGNSTOGROUP"/>
-			<xs:enumeration value="IFCRELCONTAINEDINSPATIALSTRUCTURE"/>
-			<xs:enumeration value="IFCRELNESTS"/>
-			<xs:enumeration value="IFCRELVOIDSELEMENT"/>
-			<xs:enumeration value="IFCRELFILLSELEMENT"/>
+			<xs:enumeration value="IFCRELAGGREGATES"/>  
+			<xs:enumeration value="IFCRELASSIGNSTOGROUP"/> 
+			<xs:enumeration value="IFCRELCONTAINEDINSPATIALSTRUCTURE"/> 
+			<xs:enumeration value="IFCRELNESTS"/> 
+			<xs:enumeration value="IFCRELVOIDSELEMENT IFCRELFILLSELEMENT"/> 
```

Comment:
Assuming this is a typo @CBenghi ? Should be 2 separate enumeration values?

Reply:
It's actually not. It was discussed in the call last Friday.
The idea is to transverse both those relations as if they were one.
Because just one of them does not get you from the window to the wall (the first step is voids).

It would perhaps be possible in the future to expand from this to a list of relations to consider.

Reply:
OK, that makes sense. I guess the alternative is to decouple from the explicit IFC relationships and rely on a map ( which will be needed any if just for this special case)